src/components/v5/common/ActionSidebar/partials/MultiSigSidebar/partials/MultiSigWidget/partials/ApprovalStep/ApprovalStep.tsx

```diff
@@ -165,6 +167,26 @@ const ApprovalStep: FC<ApprovalStepProps> = ({
     (signature) => signature.vote === MultiSigVote.Reject,
   ).length;
 
+  // @TODO: Leaving this here in case it is useful
```

Comment:
I support this, since it's a live branch, it's nice to keep it in until we merge the feature branch, you never know when you need these 20 lines of code :D